src/clarity-angular/layout/tabs/tab-link.directive.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2017 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import {Directive, HostListener, Inject, Input} from "@angular/core";
+import {IF_ACTIVE_ID, IfActiveService} from "../../utils/conditional/if-active.service";
+
+let nbTabLinkComponents: number = 0;
+
+@Directive({
+    selector: "[clrTabLink]",
+    host: {
+        "[id]": "tabLinkId",
+        "[attr.aria-selected]": "active",
+        "[attr.aria-controls]" : "ariaControls",
+        "role": "presentation",
```

Comment:
yeah so the ones with the `[]` treat the right side as an input that needs to be evaluated. so the id for example, is set to the value of `tabLinkId` variable. The role is just for a11y and the hard-coded value of `presentation` is assigned.